libafl_targets/src/libfuzzer.c

```diff
@@ -33,6 +37,11 @@ int main(int argc, char** argv) {
 
 #pragma GCC diagnostic pop
 
+// take a page out of libfuzzer's book: static define __sancov_lowest_stack
+// since we don't support it yet
+// TODO support it
+MAYBE_THREAD_LOCAL uintptr_t __sancov_lowest_stack;
```

Comment:
Support can be as easy as exposing this value to rust via an `fn` - then we can build an observer that observes stack depth of a run, quasi for free